azure-pipelines.yml

```diff
@@ -205,9 +198,13 @@ stages:
     - script: python runtests.py -g --refguide-check
       displayName: 'Run Refguide Check'
       condition: eq(variables['USE_OPENBLAS'], '1')
-    - script: python runtests.py -n --mode=full -- -rsx --junitxml=junit/test-results.xml
+    - script: |
+        echo LIBRARY_PATH ${LIBRARY_PATH}
+        python runtests.py -n --mode=full -- -rsx --junitxml=junit/test-results.xml
       displayName: 'Run Full NumPy Test Suite'
       condition: eq(variables['USE_OPENBLAS'], '1')
+      env:
+        LIBRARY_PATH: /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/lib
```

Comment:
How about above this line adding:
```bash
# The Python install in GitHub Actions adds `-lSystem`, which fails without adding this (see gh-22043)
```

Reply:
I don't think it is coming from the macOS-provided python, rather from gcc/gfortran (as opposed to clang, which does not add it). I did not find SHLIBS in NumPy nor in distutils/setuptools, and the stack overflows I linked to did not mention python. Adding a comment in that vein.

Reply:
makes sense. that's a gfortran that anyway needs an update soon, so hopefully that goes away in the next update